### ISSUE TITLE
Problema inserimento persone

### ISSUE THREAD

**Issue by Petrillo96** [italia/design-scuole-wordpress-theme#677]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Descrivi il problema

Buongiorno, 
sto avendo difficoltà nell'inserimento e nel popolare la sezione "Le persone". Ho già creato l'organizzazione, provato ad inserire utenti dandogli specificatamente dei ruoli ma senza alcun riscontro positivo. 
Perchè notando le versioni precedenti bastava solamente caricare l'organizzazione e poi faceva da se.
Cosa sto sbagliando?

### Includi link o screenshot

https://www.iccapoluogobrogna.edu.it

### Su quale sisitema operativo stai riscontrando il problema?

Windows

### Su quale browser stai riscontrando il problema?

Qualsiasi

### Che versione del tema stai utilizzando?

2.10.0

**Comment by dgmorgagni** [comment]
Ciao @Petrillo96 prova a guardare in "_utenti/persone_" e vedi se "_protezione privacy_" è su "_no_"
![Screenshot_3](https://github.com/italia/design-scuole-wordpress-theme/assets/79083735/9b6501c1-8faa-40e3-8c79-b3608d108798)
Dalla versione 2.10.0 è impostata su "si" di default e per far vedere l'elenco delle persone deve essere impostata su "no"

**Comment by Petrillo96** [comment]
Ciao @dgmorgagni e grazie per la risposta, ho effettuato la modifica che mi hai consigliato ed effettivamente la pagina si popola però il footer appare dimezzato rispetto alle altre pagina. Allego foto per chiarimenti.
![Firefox_Screenshot_2024-05-21T14-52-17 484Z](https://github.com/italia/design-scuole-wordpress-theme/assets/125658869/401722be-7488-4f9f-bb15-505c8138e77a)

**Comment by Petrillo96** [comment]
Ciao, ho effettuato la modifica e risulta popolarsi. Però adesso la pagina va in bug ma non riesco a capire quale sia il problema. Allego immagine.
![Firefox_Screenshot_2024-05-21T14-52-17 484Z](https://github.com/italia/design-scuole-wordpress-theme/assets/125658869/76ce8023-df73-4926-9b78-d393624df21d)

**Comment by astagi** [comment]
@Petrillo96 che tipo di problema riscontri? non visualizzi il nome?

**Comment by Petrillo96** [comment]
@astagi no, perdonami. Il footer non lo visualizzo correttamente. Cioè non è "esteso" come il resto

**Comment by astagi** [comment]
@Petrillo96 non riesco a replicare il problema, da template aggiornato vedo il footer esteso correttamente. Puoi vedere se ci sono errori nel codice html generato? può essere anche qualche regola custom del css che per qualche motivo restringe il footer sul vostro sito?

<img width="1432" alt="image" src="https://github.com/italia/design-scuole-wordpress-theme/assets/537363/981fce5d-e4bc-4e7d-88d3-2e42d2ea5010">

**Comment by Petrillo96** [comment]
@astagi al momento ho utilizzando un page builder risolvendo la cosa. Controllo il codice come mi consigli, anche perchè ho avuto difficoltà nell'aggiornare il tema alla versione recente, cosa mai accaduta fin'ora.

**Comment by astagi** [comment]
Grazie! Che problemi hai riscontrato nell'aggiornamento?

**Comment by Petrillo96** [comment]
@astagi mi sballa tutto il sito non riuscendo ad aggiornarlo. Devo per forza di cose passare alla versione precedente.

**Comment by Petrillo96** [comment]
@astagi comunque non sono riuscito a trovare l'errore  in merito al footer. Mi da lo stesso problema su ogni sito
